chainerx_cc/chainerx/python/routines.cc

```diff
@@ -277,6 +278,17 @@ void InitChainerxCreation(pybind11::module& m) {
           "device"_a = nullptr);
 }
 
+void InitChainerxEvaluation(pybind11::module& m) {
+    // evaluation routines
+    m.def("accuracy",
```

Comment:
Could you add docs to this routine (in `chainerx/_docs/routines.py`)?

Reply:
Fixed.